.github/workflows/ci.yml

```diff
@@ -667,7 +667,7 @@ jobs:
     - name: package installs
       run: |
         sudo apt-get update
-        sudo apt-get -yq install meson pkg-config gnutls-bin libnss3-tools libnss3-dev libsofthsm2 opensc expect
+        sudo apt-get -yq install meson pkg-config gnutls-bin libnss3-tools libnss3-dev libsofthsm2 opensc expect libssl-dev
```

Comment:
Hmm... this is a strange requirement. Won't this mean it will be built against the system openssl and not the one we built?

Reply:
yeah, that was  a screw up on my part (sort of), but yeah, the rust crate that pyca-crypto now builds requires the system ossl library, whcih will be backlevel from what we're testing, but it seems like it still honors the LD_LIBRARY_PATH set in the test wrapper, so I'm not fully sure whats going on there.

Reply:
The test is passing now, and (I think) the library linkage is resolved properly to the version of libcrypto under test (please see the test results below to help confirm)

Reply:
Why are we changing `external-tests-providers` when this PR is supposed to be about pyca crypto?

Reply:
Because I'm a dummy and changed the wrong line, duh :)